test/validation/link-validator.test.js

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import { expect } from '@esm-bundle/chai';
+import {
+  getLinksLists,
+  compareLinkLists,
+  LINKS_MATCH,
+  LINKS_DO_NOT_MATCH,
+  LENGTHS_DO_NOT_MATCH,
+  validateMigratedPageLinks,
+} from '../../validation/link-validator.js';
+import { getMdast } from '../../bulk-update/document-manager/document-manager.js';
+import { ExcelReporter } from '../../bulk-update/index.js';
+
+describe('Validator', () => {
+  const sourceMd = fs.readFileSync('test/validation/mocks/adobe-experience-manager-source.md', 'utf-8');
+  const updatedMd = fs.readFileSync('test/validation/mocks/adobe-experience-manager-updated.md', 'utf-8');
+  const mismatchMd = fs.readFileSync('test/validation/mocks/adobe-experience-manager-updated-mismatched.md', 'utf-8');
+  const shuffledMd = fs.readFileSync('test/validation/mocks/adobe-experience-manager-shuffled.md', 'utf-8');
```

Comment:
Looks like the tests are failing with "File does not exist at provided path"
You need to use something like `import.meta.url` to get the full path .
```suggestion
const { pathname } = new URL('.', import.meta.url);
describe('Validator', () => {
  const sourceMd = fs.readFileSync(`${pathname}mocks/adobe-experience-manager-source.md`, 'utf-8');
  const updatedMd = fs.readFileSync(`${pathname}mocks/adobe-experience-manager-updated.md`, 'utf-8');
  const mismatchMd = fs.readFileSync(`${pathname}mocks/adobe-experience-manager-updated-mismatched.md`, 'utf-8');
  const shuffledMd = fs.readFileSync(`${pathname}mocks/adobe-experience-manager-shuffled.md`, 'utf-8');
```